src/main/java/net/sky/pieces/Piece.java

```diff
@@ -0,0 +1,73 @@
+package net.sky.pieces;
+
+public class Piece {
+
+    private final Color color;
+    private final PieceType pieceType;
+    private final char REPRESENTATION;
```

Comment:
인스턴스 변수의 네이밍 컨벤션은 **소문자**입니다.
```suggestion
    private final char representation;

Reply:
상수랑 인스턴스 변수랑 혼동을 한 것 같네요. 생각할 수 있게 해주셔서 감사합니다! 계속 변수 네이밍을 이상하게 할 뻔했네요.